helpdesk/helpdesk/doctype/hd_ticket/hd_ticket.py

```diff
@@ -534,6 +522,8 @@ def reply_via_agent(
                 subject=subject,
                 template=template,
                 with_container=False,
+                in_reply_to=last_communication.name,
+                message_id=last_communication.message_id,
```

Comment:
This doesn't seem correct. 

messaeg IDs ae unique, and in-reply-to is also a message id, not some name.